src/collection.rs

```diff
@@ -323,9 +329,64 @@ impl Collection {
         Ok(())
     }
 
+    /// Drop Partitions
+    pub async fn drop_partition<S: AsRef<str>>(&self, name: S) -> Result<()> {
+        status_to_result(&Some(
+            self.client
+                .clone()
+                .drop_partition(DropPartitionRequest {
+                    base: Some(new_msg(MsgType::DropPartition)),
+                    db_name: "".to_string(),
+                    collection_name: self.schema().name.to_string(),
+                    partition_name: name.as_ref().to_owned(),
+                })
+                .await?
+                .into_inner(),
+        ))
+    }
+
+    /// Load Partitions
+    pub async fn load_partitions<S: ToString, I: IntoIterator<Item = S>>(
+        &self,
+        names: I,
+    ) -> Result<()> {
+        status_to_result(&Some(
+            self.client
+                .clone()
+                .load_partitions(LoadPartitionsRequest {
+                    base: Some(new_msg(MsgType::LoadPartitions)),
+                    db_name: "".to_string(),
+                    collection_name: self.schema().name.to_string(),
+                    partition_names: names.into_iter().map(|x| x.to_string()).collect(),
+                    replica_number: 1,
+                })
+                .await?
+                .into_inner(),
+        ))
```

Comment:
we have to wait the load done, `Load_Xxx` method of Milvus only commits the request, use `get_loading_progress` to make sure the collection/partitions available